content/docs/studio/user-guide/views/create-view.md

```diff
@@ -3,16 +3,33 @@
 To create a new view, follow these steps.
 
 1. Sign in to your [Iterative Studio](https://studio.iterative.ai/) dashboard
-   using GitHub, GitLab or Bitbucket.
+   using your GitHub.com, GitLab.com or Bitbucket.org account, or your email
+   address.
+
+   - If you sign up using your email address, you will will an email with a link
+     to verify your email address. You will need to verify your email address
+     before you can use Studio.
+
+   - Upon successful sign-in, the Iterative Studio views dashboard opens.
 
 2. Click on `Add a View`. All the organizations that you have access to will be
    listed.
 
+> If you do not see your desired organizations or Git repositories, make sure
+> that
+> [the connection to your Git server has been set up](/doc/studio/user-guide/account-management#git-integrations).
+
 > To create views from your GitHub repositories, you must install the Iterative
 > Studio GitHub app. Refer to the section on
 > [GitHub app installation](/doc/studio/user-guide/install-github-app) for more
 > details.
 
+> To create views from repositories on your self-hosted GitLab server, you must
```

Comment:
tabs can be used here as well.

Reply:
Same as above: There are special instructions for Bitbucket, and nor for GitLab.com. The special instructions are only for GitHub and for self-hosted GitLab. For this reason (because only a subset of all cases have the special instructions), I'm not creating tabs. I've changed them to <admon>.